examples/run_torch_onnx.py

```diff
@@ -0,0 +1,163 @@
+    input_std = np.array(model.default_cfg["std"])
+
+    # Load image
+    image = cv2.imread("tests/assets/cheetah_person.jpg")
+    image = cv2.resize(image, dsize=input_size)
+    image = cv2.cvtColor(image, code=cv2.COLOR_BGR2RGB)
+    image_norm = ((image/255.0 - input_mean)/input_std).astype(np.float32)
+    image_norm = image_norm.transpose((2, 0, 1))  # HxWxC -> CxHxW
+    image_norm = image_norm[None, :]  # CxHxW -> 1xCxHxW
+
+    # Torch model inference
+    model.eval()
+    with torch.no_grad():
+        logits = model(torch.from_numpy(image_norm))
+        probs = torch.softmax(logits, dim=-1)  # BxC
+        label = probs.argmax(dim=-1)[0]
+    logger.info(f"Torch model prediction: classes ({probs.shape[-1]}) -> label ({label}) -> prob ({probs[0, label]})")
+
+    # Insert XAI head
+    xai_model: torch.nn.Module = insert_xai(model, Task.CLASSIFICATION)
+
+    # Torch XAI model inference
+    xai_model.eval()
+    with torch.no_grad():
+        outputs = xai_model(torch.from_numpy(image_norm))
+        logits = outputs["prediction"]  # BxC
+        saliency_maps = outputs["saliency_map"]  # BxCxhxw
+        probs = torch.softmax(logits, dim=-1)
+        label = probs.argmax(dim=-1)[0]
+    logger.info(f"Torch XAI model prediction: classes ({probs.shape[-1]}) -> label ({label}) -> prob ({probs[0, label]})")
+
+    # Torch XAI model saliency map
+    saliency_maps = saliency_maps.numpy(force=True).squeeze(0)  # Cxhxw
+    mask = saliency_maps[label]  # hxw mask for the label
+    mask = colormap(mask[None, :])  # 1xhxw
+    mask = cv2.resize(mask.squeeze(0), dsize=input_size)  # HxW
+    result = overlay(mask, image)
+    result = cv2.cvtColor(result, code=cv2.COLOR_RGB2BGR)
+    result_path = Path(args.output_dir) / "xai-torch.png"
+    result_path.parent.mkdir(parents=True, exist_ok=True)
+    cv2.imwrite(result_path, result)
+    logger.info(f"Torch XAI model saliency map: {result_path}")
+
+    # OpenVINO model conversion
+    ov_model = ov.convert_model(
+        xai_model,
+        example_input=torch.from_numpy(image_norm),
+        input=(ov.PartialShape([-1, *image_norm.shape[1:]],))
+    )
+    model_path = Path(args.output_dir) / "model.xml"
+    model_path.parent.mkdir(parents=True, exist_ok=True)
+    ov.save_model(ov_model, model_path)
+    logger.info(f"OpenVINO XAI model: {model_path}")
+
+    # OpenVINO XAI model inference
+    ov_model = ov.Core().compile_model(ov_model, device_name="CPU")
+    outputs = ov_model(image_norm)
+    logits = outputs["prediction"]  # BxC
+    saliency_maps = outputs["saliency_map"]  # BxCxhxw
+    probs = softmax(logits)
+    label = probs.argmax(axis=-1)[0]
+    logger.info(f"OpenVINO XAI model prediction: classes ({probs.shape[-1]}) -> label ({label}) -> prob ({probs[0, label]})")
+
+    # OpenVINO XAI model saliency map
+    saliency_maps = saliency_maps.squeeze(0)  # Cxhxw
+    mask = saliency_maps[label]  # hxw mask for the label
```

Comment:
why `mask`?

Reply:
Refined in https://github.com/openvinotoolkit/openvino_xai/pull/66/commits/44426944a60f027ab3978c9c5d4a45827e2c5aee